pkg/virtctl/imageupload/imageupload.go

```diff
@@ -116,7 +116,7 @@ func NewImageUploadCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 
 func usage() string {
 	usage := `  # Upload a local disk image to a newly created PersistentVolumeClaim:
-	virtctl image-upload --upload-proxy-url=https://cdi-uploadproxy.mycluster.com --pvc-name=upload-pvc --pvc-size=10Gi --image-path=/images/fedora28.qcow2`
+	virtctl image-upload --uploadproxy-url=https://cdi-uploadproxy.mycluster.com --pvc-name=upload-pvc --pvc-size=10Gi --image-path=/images/fedora28.qcow2`
```

Comment:
This really a typo?  The container/service is `cdi-uploadproxy`.

Reply:
It doesn't match the actual parameter which is `--uploadproxy-url`

Reply:
Ha I was seeing reverse.  Sorry.  My brain is clearly turned off and in vacation mode.  No more githubbing until the 26th.